CHANGES.md

```diff
@@ -76,6 +76,7 @@ To be released.
  -  Added `NamespaceNotFoundException` class.  [[#232]]
  -  Added `SimultaneousTxsException` class.  [[#242]]
  -  Added `Block<T>.Evaluate()` method.  [[#243]]
+ -  Updated access modifier of `InvalidBlockTimestampException` to public [[#133], [#251]]
```

Comment:
```suggestion
 -  Made `InvalidBlockTimestampException` class `public` so that it can be
    caught.  [[#133], [#251]]
```

Reply:
`[#133]` and `[#251]` labels also should have their reference definitions.  (See also Markdown's [reference links spec][1].)

[1]: https://spec.commonmark.org/0.29/#reference-link

Reply:
I updated the line and defined references